docs/content/latest/secure/authentication/host-based-authentication.md

```diff
@@ -25,54 +25,70 @@ showAsideToc: true
+
+{{< note title="Tip" >}}
+
+To connect to a particular database, a user must not only pass the `--ysql_hba_conf_csv` checks, but must have the `CONNECT` privilege for the database. To restrict which users can connect to which databases, granting or revoking the `CONNECT` privilege is typically easier than putting the rules in `--ysql_hba_conf_csv` entries. Refer to the [`GRANT`](../../../api/ysql/the-sql-language/statements/dcl_grant/) and [`REVOKE`](../../../api/ysql/the-sql-language/statements/dcl_revoke/) YSQL statements.
+
+{{< /note >}}
+
+## ysql_hba.conf file
+
+Records in the YugabyteDB `ysql_hba.conf` file are autogenerated based on the values included in the `--ysql_hba_conf_csv` flag. For example, starting a YB-TServer with the following `--ysql_hba_conf_csv` flag will enable MD5 authorization for all users except `yugabyte`, which can use trust when connecting from localhost:
+
+```sh
+--ysql_hba_conf_csv="host all yugabyte 127.0.0.1/0 trust, host all all 0.0.0.0/0 md5, host all yugabyte ::1/128 trust, host all all ::0/0 md5"
 ```
 
-To display the current values in the `ysql_hba.conf` file, run the following `SHOW` statement to get the file location:
+To display the values in the `ysql_hba.conf` file, run the following `SHOW` statement to get the file location:
 
-```plpgsql
+```sql
 yugabyte=# SHOW hba_file;
 ```
 
-```
+```output
                              hba_file
 -------------------------------------------------------------------
  /Users/yugabyte/yugabyte-data/node-1/disk-1/pg_data/ysql_hba.conf
 (1 row)
 ```
 
-and then view the file. Because the file is autogenerated, edits are overwritten by the autogenerated content.
-
-The `ysql_hba.conf` file includes the records autogenerated based on your [`--ysql_hba_conf`](../../../reference/configuration/yb-tserver/#ysql-hba-conf) flag. Here is an example.
+and then view the file. Because the file is autogenerated, edits are overwritten by the autogenerated content. Here is an example.
 
-```
+```output
 # This is an autogenerated file, do not edit manually!
-host all all 0.0.0.0/0 trust
-host all all ::0/0 trust
+host all yugabyte 127.0.0.1/0 trust
+host all yugabyte ::1/128 trust
 /Users/yugabyte/yugabyte-data/node-1/disk-1/pg_data/ysql_hba.conf (END)
 ```
 
-## Record formats
+Because `ysql_hba.conf` records are examined sequentially for each connection attempt, the order of the records is significant. Typically, earlier records will have tight connection match parameters and weaker authentication methods, while later records will have looser match parameters and stronger authentication methods. For example, you might want to use trust authentication for local TCP/IP connections, but require a password for remote TCP/IP connections. In this case, a record specifying `trust` authentication for connections from `127.0.0.1` would appear before a record specifying password authentication for a wider range of allowed client IP addresses.
```

Comment:
Typically, earlier records will have tight connection match parameters and weaker authentication methods, while later records will have looser match parameters and stronger authentication methods. --> Typically, earlier records have tight connection match parameters and weaker authentication methods, while later records have looser match parameters and stronger authentication methods.